src/main/RP.ts

```diff
@@ -50,7 +48,7 @@ export class RP {
   }
 
   public async verifyAuthenticationResponseJwt(
-    jwt: string,
+    payload: AuthenticationResponsePayload,
```

Comment:
This doesn't make sense to me. It was a string, which a JWT should be in the end.

Reply:
Fixed